modmesh/core.py

```diff
@@ -105,6 +107,34 @@ def _load():
     for name in __all__:
         globals()[name] = getattr(_impl, name)
 
+    # Try to find the thirdparty, if failed to find it
+    # modmesh will raise ImportError and remind the user.
+    filename = os.path.join('thirdparty')
+    path = os.getcwd()
+    try:
+        while True:
+            if os.path.exists(os.path.join(path, filename)):
+                break
+            if path == os.path.dirname(path):
+                path = None
+                break
+            else:
+                path = os.path.dirname(path)
+        if path is None or not os.path.exists(os.path.join(path,
+                                                           filename)):
+            raise ImportError
+    except ImportError:
+        sys.stderr.write('Can not find thirdparty.\n')
+        sys.stderr.write('No python 3rd-party library will be imported\n')
```

Comment:
Do you need the trailing `\n`?

Reply:
Yes, without tailing `\n`, the message on the screen will become:
```
Can not find thirdparty.No python 3rd-party library will be imported
```

Reply:
I meant the `\n` in `... imported\n`.

Reply:
Oh, it still needs `\n` after `....import`, without `\n`, the shell prompt will not start on a  newline:
```
Can not find thirdparty.
No python 3rd-party library will be imported$
```
it will not look not good.

Reply:
I see.  Thanks.